docs/advanced/custom-components.md

```diff
@@ -52,6 +52,9 @@ Stop propagation of click event to the map component. Can be used to stop map fr
 ##### `captureDoubleClick` {Boolean} - default: `true`
 Stop propagation of dblclick event to the map component. Can be used to stop map from zooming when this component is double clicked.
 
+##### `capturePointerMove` {Boolean} - default: `false`
+Stop propagation of pointermove event to the map component.  Can be used to stop map from calling the `onHover` callback when this component is clicked.
```

Comment:
The callbacks in this library are called `onMouseMove` and `onTouchMove`.

Reply:
Hopefully should be ok now.

Reply:
when this component is <del>clicked</del> hovered

Reply:
That's been changed now